journal-entry/journal-entry.qml

```diff
@@ -165,8 +165,31 @@ QtObject {
             }
 
             // Create the new journal note.
-            script.createNote(headline + "\n" + '='.repeat(Math.max(4, headline.length)) + "\n\n" + noteBodyTemplate);
-            
+            // Create the new journal note without the filename and empty line.
+            // Create the new journal note with only the body template content - code line shortened by ChatGPT.
+            script.createNote(noteBodyTemplate);
```

Comment:
There are three descriptions for this line now. Which one of them is the right one?

Reply:
And you removed the headline, was this intentional?

Reply:
lines 167 and 168 are the original ones, and should be deleted.

lines 168 and 169 in effect are the same. So, we could rephrase it as:
`// Create the new journal note with only the body template content (= without the filename and empty line).`

Removing the headline was intentional because I don't want the file name of the note repeated in the note itself, I never do that. Now this is something particular to me, and I understand you may want to keep it. In that case I will use my version of the `.qml` file.

Reply:
You marked this as resolved, but you didn't change anything. Was this an accident?

Reply:
No, it was not an accident, I thought I was done. So what is required now?

Reply:
https://github.com/qownnotes/scripts/pull/251#discussion_r1890205259

Reply:
For the 2nd part: If you don't want to / can't create a setting, please add the removed code again.